README.md

```diff
@@ -30,6 +30,7 @@ MSP customers gathering logs from linked accounts should create an **Accounts AP
 - Make sure you are running Python 3+ with `python --version`.
 - Clone this GitHub repository and navigate to the `duo_log_sync` folder.
 - Ensure you have "setuptools" by running `pip3 install setuptools`.
+- Install required packages by running `pip/pip3 install -r requirements.txt`.
```

Comment:
Would it also be useful to strongly recommend using a python virtual environment? Sometimes things can get wonky if you have multiple python projects with different dependency versions, using a single global python environment. If you give me a moment, i can provide instructions for that

Reply:
Running in a virtual environment could become problematic for those servers that run DLS as a scheduled task (Windows) or have automatic startup configured via systemd or some alternate *nix platform method. If it is assumed that there is a dedicated server deployed for DLS, it should be safe to recommend that all required packages be installed for global system usage.

Reply:
I don't know how DLS would run so I'm not sure if that assumption of running on its own server is correct? I'm okay with excluding the virtualenv instructions though.